app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Api::V1::CommentsController < ApplicationController
+  def index
```

Comment:
- [ ] @team Great work on your API endpoints, however, the response from the comments get and posts from the URL provided is not JSON 
![image](https://github.com/Munyabelden/Blog_App/assets/61408860/df338df6-4740-4b43-bf91-4613bda7ca0d)